File: generate-text/open-ended.js
import { TwelveLabs } from 'twelvelabs-js';

(async () => {
  const client = new TwelveLabs({ apiKey: process.env.API_KEY}); 

  const result = await client.generate.text(
    "66ebe4bfd4f452b062c63e3a",
    "Be concise",
    0.7
  );
  console.log(`Result ID: ${result.id}`);
  console.log(`Generated text: ${result.data}`);
})();

File: generate-text/open-ended.py
from twelvelabs import TwelveLabs
import os

client = TwelveLabs(api_key=os.environ.get("API_KEY"))

result = client.generate.text(
    video_id="66ebe4bfd4f452b062c63e3a", prompt="Be concise", temperature=0.7
)

print("Result ID:", result.id)
print(f"Generated Text: {result.data}")
